app/controllers/api/users.php

```diff
@@ -1816,8 +1817,11 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $countryName = $locale->getText('countries.' . strtolower($session->getAttribute('countryCode')), $locale->getText('locale.country.unknown'));
 
         $session = $dbForProject->createDocument('sessions', $session);
+
+        $dbForProject->purgeCachedDocument('users', $user->getId());
+
         $session
-            ->setAttribute('secret', $secret)
```

Comment:
We have to document this right, it's sort of breaking? Some people could be relying on it

Reply:
I don't think anyone is using it because it doesn't work.